vms/shift/templates/shift/cancel_email.txt

```diff
@@ -0,0 +1,3 @@
+{% autoescape off %}
+{{admin_first_name}} {{admin_last_name}} has cancelled your shift starting from {{shift_start_time}} to {{shift_end_time}} of {{job_name}} under {{event_name}} event. For any questions or concerns, email {{admin_email}}.
```

Comment:
shouldn't it  be `during the {{event_name}} event`?

Reply:
You should add date too.

Reply:
done @abhishekarya286 Please review